services/etcd/config.go

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package client
+package etcd
```

Comment:
Should the package be `services/client/etcd` instead? This way if in the future there's a non-etcd implementation, it'll go under `services/client/` as well?

Reply:
sure can do